endpoint.go

```diff
@@ -583,6 +586,17 @@ func (ep *endpoint) Statistics() (map[string]*sandbox.InterfaceStatistics, error
 	return m, nil
 }
 
+func (ep *endpoint) SandboxInterfaces() []sandbox.Interface {
```

Comment:
This would return all the interfaces in the sandbox, also the one connecting to other endpoints.
Since this function belongs to Endpoint API, I'd expect it to return only the interface owned by this Endpoint.

If the goal instead is to get all the interfaces on the sandbox, I think we should wait for #365 to be merged and add the new API to the Sandbox interface.